src/main/java/org/sagebionetworks/web/client/place/ACTDataAccessSubmissionsPlace.java

```diff
@@ -7,8 +7,7 @@
 
 	public static final String ACCESS_REQUIREMENT_ID_PARAM = "AR_ID";
 	public static final String STATE_FILTER_PARAM = "STATE";
-	public static final String MIN_DATE_PARAM = "FROM_DATE";
```

Comment:
Deleted dead code (not sure if this date filter was something that we talked about implemented but never got around to it?).